packages/react/src/profiler.tsx

```diff
@@ -39,14 +39,34 @@ function afterNextFrame(callback: Function): void {
   timeout = window.setTimeout(done, 100);
 }
 
+/**
+ * isProfilingModeOn tells us if the React.Profiler will correctly
+ * Profile it's components. This is only active in development mode
+ * and in profiling mode.
+ *
+ * Learn how to do that here: https://gist.github.com/bvaughn/25e6233aeb1b4f0cdb8d8366e54a3977
+ */
+function isProfilingModeOn(): boolean {
+  const fake = React.createElement('div') as any;
+
+  // tslint:disable-next-line: triple-equals no-unsafe-any
+  if (fake._owner != null && fake._owner.actualDuration != null) {
```

Comment:
This sucks, but I can't find a better way to figure out if a user can profile or not.